### ISSUE TITLE
suggestion-line (the 6th row) does not work anymore

### ISSUE THREAD

**Issue by DerHelmut** [klausw/hackerskeyboard#537]
In Hackers Keyboard the suggestion-line (the 6th row) does not work anymore at my Acer Liquid Z630S with Android 6: it shows the symbols and even the suggestions like it used to, but it´s not possible anymore to klick on it.
Installing new didn´t help and it did work perfekt before for 3 month...

So sad, that makes writing much slower...

Anyone have an idea?

**Comment by PakoPakoJr** [comment]
I have the same issue; the only suggestions that appear are pulled from my persknal Dictiknary.
Also, touhc to auto-correct has been disabled, even jf selected from the settings menu; some apps like Google Docs have thjs feature built in, but skme like Chrome, dk not.

**Comment by DerHelmut** [comment]
I think, my problem´s a little different: 
suggestions do come very well and are shown in the suggestion-line, but if I tap at them, nothing happens. Only when I long-press on a suggestion, it will appear in the text

**Comment by dadecamp** [comment]
Im having same issue. No 6th row with sugestion showing.

**Comment by DerHelmut** [comment]
@dadecamp: no, that´s not the same issue: in my case there IS a 6th row, it´s just not clickable!

You should maybe just look into the settings...
